### PR TITLE
Issue 74: Removed url from tip record

### PR DESCRIPTION
**Pull request by stanislav-slavov**

### CODE REVIEW

File: src/components/tipRecords/TipRecordComponent.vue [hunk 1]
```diff
@@ -14,9 +14,6 @@
           <tip-title :tip="tip" :goToTip="goToTip"></tip-title>
         </div>
       </div>
-      <div>
```
Comment (thepiwo) [inline src/components/tipRecords/TipRecordComponent.vue]: as this is removed, whats linking to the url now?

File: src/components/tipRecords/TipRecordComponent.vue [hunk 2]
```diff
@@ -31,7 +31,9 @@
         </div>
       </div>
     </div>
-    <div class="tip__article col-4" v-if="isPreviewToBeVisualized(tip)">
+    <div class="tip__article col-4 position-relative" v-if="isPreviewToBeVisualized(tip)">
```
Comment (thepiwo) [inline src/components/tipRecords/TipRecordComponent.vue]: `isPreviewToBeVisualized` also within this?

Comment (stanislav-slavov) [reply src/components/tipRecords/TipRecordComponent.vue]: These are minor changes depending on one another. That's why I combined it.

Comment (thepiwo) [reply src/components/tipRecords/TipRecordComponent.vue]: but the if isPreviewToBeVisualized is also nested inside which doesn't make sense to me

Comment (stanislav-slavov) [reply src/components/tipRecords/TipRecordComponent.vue]: I caught your point, the section should be hidden if there are no results. Only one check will stay

**Pull request merged**